packages/common/src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export type Transaction = {
+  id?: string;
+  customer: string;
+  supplier: string;
+  total: string;
+  date: string;
+};
```

Comment:
Just starting the `common/types` - likely will need two types one for the list view and one for the detail, so we don't have so many optionals